shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -174,7 +211,9 @@ case class Arrival(Operator: Option[Operator],
       Stand = if (incoming.Stand.exists(_.nonEmpty)) incoming.Stand else this.Stand,
       Gate = if (incoming.Gate.exists(_.nonEmpty)) incoming.Gate else this.Gate,
       RedListPax = if (incoming.RedListPax.nonEmpty) incoming.RedListPax else this.RedListPax,
+      TotalPax = if (incoming.TotalPax.nonEmpty) this.TotalPax ++ incoming.TotalPax else this.TotalPax,
```

Comment:
This will do the wrong thing. We'll end up with multiple numbers from the same source. Let's write a test for it to make sure it behaves the way we expect it to